src/functions-and-arrays.js

```diff
@@ -1,19 +1,49 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(a, b) {
+  if (a >= b) {
+    return a 
+  } else {
+    return b
+  }
+}
 
 
 
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words = [];
+
+function findLongestWord(words) {
+  // console.log(words)
+  let longestWord=''
+  if (words.length === 0) {
+   
+    return null;
+  }
```

Comment:
For shorter syntax,  if you return only on line after if statement, you could write it like this:

```
if (words.length === 0) return null;
```